geoapps/inversion/components/models.py

```diff
@@ -114,71 +114,73 @@ def active_cells(self, active_cells):
         self._active_cells = active_cells
 
     @property
-    def starting(self):
-        mstart = self._starting.model
+    def starting(self) -> np.ndarray | None:
+        if self._starting.model is None:
+            return None
+
+        mstart = self._starting.model.copy()
 
         if mstart is not None and self.is_sigma:
-            mstart = mstart.copy()
             mstart = np.log(mstart)
 
         return mstart
 
     @property
-    def reference(self):
+    def reference(self) -> np.ndarray | None:
         mref = self._reference.model
```

Comment:
The logic in here is complicated.  Is there a way to combine the `if mref is None` and `if self.is_sigma & (all(mref == 0))`  and can we stick with one name for reference model?

Reply:
You got it.